README.md

```diff
@@ -20,9 +20,7 @@ bundle install
 touch .env
 
 # Clone production data
-rake db:create db:migrate
-rake cascade:sync
-rake faculty:sync
+cap production db:clone
```

Comment:
I worry this is going to create confusion in the future. `cap == deployment` in my mind. If you can point to something that suggests that this is a faulty or oversimplified association, I'll adjust.

I feel like the interface for this task should be:

    rake db:clone production